api/group/repository/addUserToGroup.js

```diff
@@ -0,0 +1,8 @@
+import { UserGroup } from '../../common/models/relations.js';
```

Comment:
I could not find api/common/models/relations.js and ./lib/db.js in your repository
Did you push that files?